project/apps/matching-service/src/db/match.redis.ts

```diff
@@ -47,6 +49,181 @@ export class MatchRedis {
 
   async getSocketByUserId(userId: string) {
     const userSocketKey = `${USER_SOCKET_KEY}-${userId}`;
-    return await this.cacheManager.get<string>(userSocketKey);
+    return await this.cacheManager.get(userSocketKey);
+  }
```

Comment:
is there a need for cacheManager anymore since we have the redisClient?

Reply:
yup that would be great, so we also dont get confused in the future of which one we should use when developing